### PR TITLE
insert privacy consent note only one

### PR DESCRIPTION
**Pull request by alikon**
Pull Request for Issue https://github.com/joomla/joomla-cms/pull/20051#issuecomment-381861440 .

### Summary of Changes
insert privacy consent note only one

@sandewt please test

### CODE REVIEW

File: plugins/user/privacyconsent/privacyconsent.php [hunk 1]
```diff
@@ -202,48 +202,56 @@ public function onUserAfterSave($data, $isNew, $result, $error)
...
+			$userId = ArrayHelper::getValue($data, 'id', 0, 'int');
 
-		// Create the consent confirmation
-		$confirm = (object) array(
-			'user_id'       => $userId,
-			'profile_key'   => 'consent',
-			'profile_value' => 1
-		);
+			// Get the user's IP address
+			$ip = $this->app->input->server->get('REMOTE_ADDR');
```
Comment (brianteeman) [inline plugins/user/privacyconsent/privacyconsent.php:213]: can you revert the change on this line please

File: plugins/user/privacyconsent/privacyconsent.php [hunk 2]
```diff
@@ -202,48 +202,56 @@ public function onUserAfterSave($data, $isNew, $result, $error)
...
 
-		try
-		{
-			$this->db->insertObject('#__user_profiles', $confirm);
-		}
-		catch (Exception $e)
-		{
-			// Do nothing if the save fails
+			// Get the user agent string
+			$user_agent = $this->app->input->server->get('HTTP_USER_AGENT');
```
Comment (brianteeman) [inline plugins/user/privacyconsent/privacyconsent.php:216]: and here

File: plugins/user/privacyconsent/privacyconsent.php [hunk 3]
```diff
@@ -202,48 +202,56 @@ public function onUserAfterSave($data, $isNew, $result, $error)
...
+
+			// Get the date in DB format
+			$now = JFactory::getDate()->toSql();
```
Comment (brianteeman) [inline plugins/user/privacyconsent/privacyconsent.php:219]: why this change

**Comment by alikon** [comment]
silly me i'm using an old branch

**Comment by alikon** [comment]
redone https://github.com/brianteeman/joomla-cms/pull/76
hope against the latest branch ;)
